libs/h3/src/main/java/org/elasticsearch/h3/Vec2d.java

```diff
@@ -29,6 +29,11 @@
  */
 final class Vec2d {
 
+    /**
+     * 2.0 * PI
+     */
+    private static final double M_2PI = 6.28318530717958647692528676655900576839433;
```

Comment:
Wouldn't the compiler do this for us anyway? I noticed you did not use this in LatLng.java.

Reply:
I added the variable to the Constants and use it in LatLng too.